### ISSUE TITLE
[Feature]: Customizable colors for token labels

### ISSUE THREAD

**Issue by Zahariel** [RPTools/maptool#4876]
### Describe the Problem

The "Token Name" labels, for any NPC token (on any layer), are really hard to read. I use these to put a name to each town and point of interest on the world map, but it's close to useless. White-on-light-blue is just not a useful color scheme; it's terrible. My players complain about it constantly.

### The Solution you'd like

Please make it possible to change the color schemes of text labels. A more useful default for NPC tokens would be nice as well.

**Comment by cwisniew** [comment]
@Zahariel  is https://github.com/RPTools/maptool/pull/4662 what you are after?

**Comment by Zahariel** [comment]
Yes, thank you! That is exactly what I was hoping for! Do you have any idea when it will be released? It looks like 1.14.3 is still the official version, which definitely doesn't have it, and I use MapTool too much to want to deal with nightly builds.

**Comment by Pmofmalasia** [comment]
You can find the release candidate for 1.15 here instead of using nightly builds: https://github.com/RPTools/maptool/releases/tag/1.15.0-rc.3

So far it seems fairly stable, but could use some more testing - if you could try it out that would be helpful! And if there's no issues that would be the release anyway so you could continue to use it.

**Comment by cwisniew** [comment]
Done in #4662
